src/applications/vaos/utils/data.js

```diff
@@ -197,42 +197,14 @@ export function transformFormToAppointment(state) {
 
   return {
     clinic,
-    direct: {
-      purpose,
-      desiredDate: moment(slot.date, 'YYYY-MM-DD').format(
-        'MM/DD/YYYY [00:00:00]',
-      ),
-      dateTime: moment(slot.datetime).format('MM/DD/YYYY HH:mm:ss'),
-      apptLength: appointmentLength,
-    },
     // These times are a lie, they're actually in local time, but the upstream
     // service expects the 0 offset.
     desiredDate: `${slot.date}T00:00:00+00:00`,
     dateTime: moment(slot.datetime).format('YYYY-MM-DD[T]HH:mm:ss[+00:00]'),
     duration: appointmentLength,
     bookingNotes: purpose,
-    patients: {
-      patient: [
```

Comment:
Same for `patient`.  API takes care of this object using just the `preferredEmail` and `timeZone`